### ISSUE TITLE
how to about custom alias?

### ISSUE THREAD

**Issue by fz420** [ohmybash/oh-my-bash#7]
```
03:45:17 root@instance-2 ~ → cat .oh-my-bash/custom/aliases/example.aliases.sh
# Add your own custom alias in the custom/aliases directory. Aliases placed
# here will override ones with the same name in the main alias directory.

alias ll='ls -l'
alias la='ll -a'
alias ld='ll -d'
alias cc='clear'

echo `dirname``basename`
03:46:56 root@instance-2 ~ → source .bashrc
03:47:04 root@instance-2 ~ → cc
cc: fatal error: no input files
compilation terminated.
03:47:05 root@instance-2 ~ →

```

**Comment by nntoan** [comment]
@fz420 Can you please try `which cc` and see what is the output?

**Comment by fz420** [comment]
@nntoan  other host
```
05:54:07 lite@arch .oh-my-bash ±|master ✗|→ cat custom/aliases/example.aliases.sh
# Add your own custom alias in the custom/aliases directory. Aliases placed
# here will override ones with the same name in the main alias directory.

alais ll='ls -l'
alias vi='vim'
alias ld='ll -d'
alias la='ll -a'
alias cc='clear'

05:54:11 lite@arch .oh-my-bash ±|master ✗|→ cc
cc: fatal error: no input files
compilation terminated.
05:54:16 lite@arch .oh-my-bash ±|master ✗|→ which cc
/usr/bin/cc
05:54:18 lite@arch .oh-my-bash ±|master ✗|→ la
aliases     CONTRIBUTING.md  .git        LICENSE.txt    plugins    themes
cache       custom           .gitignore  log            README.md  tools
completion  .editorconfig    lib         oh-my-bash.sh  templates
05:54:47 lite@arch .oh-my-bash ±|master ✗|→

```

**Comment by nntoan** [comment]
@fz420 How about the other aliases? For e.g, try `alias vis="vim"` and check if it work?

**Comment by fz420** [comment]
@nntoan 
```
[c:\~]$

Connecting to 192.168.37.140:22...
Connection established.
To escape to local shell, press 'Ctrl+Alt+]'.

Last login: Tue Mar 27 14:29:09 2018 from 192.168.37.1
which: no aws_completer in (/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
which: no register-python-argcomplete in (/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
which: no jungle in (/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
which: no kontena in (/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
02:29:50 lite@arch ~ → tail .oh-my-bash/custom/aliases/example.aliases.sh
# Add your own custom alias in the custom/aliases directory. Aliases placed
# here will override ones with the same name in the main alias directory.

alais ll='ls -l'
alias vi='vim'
alias ld='ll -d'
alias la='ll -a'
alias cc='clear'

alias vis='vim'
02:29:57 lite@arch ~ → vis
-bash: vis: command not found
02:30:03 lite@arch ~ →


```

**Comment by tmidi** [comment]
This is the current `clear` alias:
`alias c='clear'                             # c:            Clear terminal display`

after changing the alias, did you try to source the new alias file:
`source .oh-my-bash/custom/aliases/example.aliases.sh`

**Comment by rubenbonilla** [comment]
@nntoan I think this issue is solved by #30 pull request

**Comment by nntoan** [comment]
@rubenbonilla That's correct! Thanks @rubenbonilla 
I'll close this ticket then.

Feel free to reopen if you see this issue once again.
Cheers
